src/components/ServiceList.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react";
+import FilterList from "../components/FilterList";
+import useServicesList from "../hooks/useServicesList";
+import ServiceCard from "../components/ServiceCard";
+
+const compare = (a, b) => {
```

Comment:
please remove this, the current work item this pr is adderssing doesn't have anything to do with sorting. If the list isnt sorting properly that should be handled in teh service.

Reply:
done